### ISSUE TITLE
If the OLD URL is end up with '/' then the redirect doesn't work

### ISSUE THREAD

**Issue by dstream** [wkallhof/Simple301#25]

**Comment by LettuceGnome** [comment]
Yep, I'm experiencing this too with Umbraco 7.6.5.  Everything works fine if the trailing slash is omitted--trouble is, our content editors aren't very good at remembering things.

**Comment by richarth** [comment]
I'm still experiencing this issue with version 1.1.4 and Umbraco 7.14.1
